common/changes/@itwin/appui-codemod/kayla-componentExamples_2023-06-29-20-20.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
`rush change` only generated this due to an (unnecessary) formatting change in codemod package. I think you should get rid of the change files, run `rush prettier:fix` then `rush change` again.